src/interface/src/app/treatments/direct-impacts/metric-filters/metric-selector/metric-selector.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  <div class="fixed-width">
+    <mat-icon
+      class="material-symbols-outlined"
+      [style.color]="
```

Comment:
similarly here- this can be just @Input to this component for the color, or even static or dictionary-driven by the position (metric 1, 2 etc)

Reply:
Now we are binding the color as an input